aws-storage-s3/src/androidTest/java/com/amplifyframework/storage/s3/AWSS3StorageDownloadTest.java

```diff
@@ -55,6 +56,7 @@
 /**
  * Instrumentation test for operational work on download.
  */
+@Ignore("Contains tests that hang, or hang the suite overall.")
```

Comment:
Did you intend this change for this PR?

Reply:
Yes; I segregated these `@Ignore`s into a separate git commit. The PR includes 2. (I needed the tests to work, in order to compute coverage.)

Reply:
Gotcha. This is a pretty major @ignore given we'll no longer have coverage for these common operations. We should probably add chores to address these when we can.

Reply:
Yup, we have a bunch of ignored tests in total.